### ISSUE TITLE
Question about the number of different blurred images in the dataset and the issue of dividing the training/test sets

### ISSUE THREAD

**Issue by wsygdsg666** [JZD151/Real-IISR#8]
Could you please clarify whether the "Optical blur" in the image annotations corresponds to the "defocus blur" mentioned in the paper? If so, this would cause inconsistency in the number of images labeled as "defocus blur" and "motion blur" in the paper. The annotations in the file only have 43 labels for "motion blur", while the paper states 152; this would have an impact on the division of the training set and test set. Please provide an explanation.
